queries.sql

```diff
@@ -7,4 +7,24 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name= 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5; 
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Update queries to remove negative weights. */
+
+BEGIN TRANSACTION;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT my_sp;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT my_sp;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT * from animals;
+
+/* Queries for finding aggregates. */
```

Comment:
- [ ] Among the transactions that you were supposed to record here, there are missing three of them. 

![image](https://user-images.githubusercontent.com/90222110/228608000-a6d4bc3b-c9dd-4acb-9997-3d8ce18c21cd.png)

![image](https://user-images.githubusercontent.com/90222110/228608218-5628d528-ada3-4fb6-8845-d441f8b58ea0.png)

![image](https://user-images.githubusercontent.com/90222110/228608372-9377ec68-2dfb-469c-a377-be01978e3b0c.png)

All these transaction steps should be recorded here too so please add them here 😃